requirements.txt

```diff
@@ -1 +1,2 @@
 pydantic==1.8.1
+snakemake==7.2.1
```

Comment:
The question is whether that is going to be required if the ZARP pipeline is installed anyway in #24. But okay to keep, just something to consider

Reply:
In commit e2eeea59691b15332e82dd4d7cdeb440298fef98, I removed the requirement, but now the tests won't pass anymore here. For me locally they pass because I still have `snakemake` within miniconda.

What is the best way to deal with this?